compiler/ast.mli

```diff
@@ -15,18 +15,25 @@ type op =    (* Operators *)
   | Div      (* / *)
   | Mod      (* % *)
   | Pow      (* ** *)
+  | Not      (* ! *)
+  | Equal    (* == *)
+  | Neq      (* != *)
+  | Less     (* < *)
+  | Leq      (* <= *)
+  | Greater  (* > *)
+  | Geq      (* >= *)
 
 type expr =                      (* Expressions *)
   | Int_lit of int               (* 42 *)
   | Float_lit of float           (* 42.0 *)
   | String_lit of string         (* "Hello, world" *)
+  | Bool_lit of bool             (* true *)
   | Id of string                 (* x *)
-  | Binop of expr * op * expr    (* a + b *)
```

Comment:
consistency with the other files. the order doesn't really matter unless it could get confused with things above it.